lib/security/waf.ts

```diff
@@ -77,6 +107,52 @@ const awsManagedRules: WafRule[] = [
       },
     },
   },
+  // AWS Managed Rules Known Bad Inputs Rule Set
+  {
+    name: 'AWS-AWSManagedRulesKnownBadInputsRuleSet',
+    rule: {
+      name: 'AWS-AWSManagedRulesKnownBadInputsRuleSet',
+      priority: 4,
```

Comment:
This needs to be priority 2

Reply:
Updated the PR with new rules. I put allowed GitHub Ip the highest priority to avoid block by other AWS managed rules in case.